packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -97,7 +99,7 @@ export class GraphQLPersonalizeService {
       return data?.layout?.item
         ? {
             // CDP expects content id format `<id>_<language>_<version>` (lowercase)
-            contentId: `${data.layout.item.id}_${language}_${data.layout.item.version}`.toLowerCase(),
+            contentId: `${EMBEDDED}${data.layout.item.id}_${language}`.toLowerCase(),
```

Comment:
And `_` is missing after `embedded`
```suggestion
            contentId: `embedded_${data.layout.item.id}_${language}`.toLowerCase(),
```